incubator/cockroachdb/templates/NOTES.txt

```diff
@@ -0,0 +1,27 @@
+CockroachDB can be accessed via port 26257 (or whatever you set the GrpcPort
+value to) at the following DNS name from within your cluster:
+{{ .Release.Name }}-public.{{ .Release.Namespace }}.svc.cluster.local
+
+Because CockroachDB supports the PostgreSQL wire protocol, you can connect to
+the cluster using any available PostgreSQL client.
+
+For example, you can open up a SQL shell to the cluster by running:
+
+    kubectl run -it --rm cockroach-client \
+        --image=cockroachdb/cockroach \
+        --restart=Never \
+        --command -- ./cockroach sql --host {{ .Release.Name }}-public.{{ .Release.Namespace }}
+
+From there, you can interact with the SQL shell as you would any other SQL shell,
+confident that any data you write will be safe and available even if parts of
+your cluster fail.
+
+Finally, to open up the CockroachDB admin UI, you can port-forward from your
+local machine into one of the instances in the cluster:
+
+    kubectl port-forward {{ .Release.Name }}.{{ .Release.Namespace }}-0 8080
```

Comment:
The pods are actually named as follows:
{{ .Release.Name }}-{{ .Chart.Name }}-0